Lib/urllib/parse.py

```diff
@@ -30,6 +30,7 @@
 import re
 import sys
 import collections
+import unicodedata
```

Comment:
Please load the module only when necessary. The unicodedata is a very large module and usually not loaded. The shared library is over 1 MB. 

```
>>> import urllib.parse, urllib.request
>>> import sys
>>> 'unicodedata' in sys.modules
False
```